src/mlpack/methods/reinforcement_learning/q_learning_impl.hpp

```diff
@@ -180,22 +159,17 @@ double QLearning<
   arma::mat target;
   learningNetwork.Forward(sampledStates, target);
 
+  double discount = pow(config.Discount(), replayMethod.NSteps());
```

Comment:
```suggestion
  double discount = std::pow(config.Discount(), replayMethod.NSteps());
```

We had some problems in the past, where the compiler couldn't figure out to use `arma::pow` or `std::pow`.